### PR TITLE
Minor documentation formatting and restructuring

### PR DESCRIPTION
**Pull request by mherwig**
I updated the README.md so that it "looks and feels" like the READMEs of all other components (only minor changes).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +32,24 @@ There's a docker-compose file included in this project which provides a Kafka, K
...
+Assuming you have already installed [go](https://go.dev/), simply run the follwoing to build the executable:
+```bash
+go build
+```
+
+> Alternatively, you can also follow the Docker build in the following section if you want to build a Docker image without the need to have Golang installed locally.
+
+### Docker build
+
+This repository provides a multi-stage Dockerfile that will also take care about compiling the software, as well as dockerizing Vortex. Simply run:
```
Comment (Th3Shadowbroker) [inline README.md:48]:
```suggestion
This repository provides a multi-stage Dockerfile that will also take care of compiling the software, as well as dockerizing Vortex. Simply run:
```

File: README.md [hunk 2]
```diff
@@ -5,13 +5,19 @@ SPDX-License-Identifier: Apache-2.0
...
 </p>
 
+<p align="center">
+  Connector used by Horizon for transferring and transforming data from Kafka to MongoDB.
+</p>
 
 <p align="center">
-  A connector for transferring and transforming data from Kafka to MongoDB.
+  <a href="#prerequisites">Prerequisites</a> •
+  <a href="#building-vortex">Building Voretex</a> •
```
Comment (Th3Shadowbroker) [inline README.md:18]:
```suggestion
  <a href="#building-vortex">Building Vortex</a> •
```

File: README.md [hunk 3]
```diff
@@ -5,13 +5,19 @@ SPDX-License-Identifier: Apache-2.0
 -->
 
 <p align="center">
-  <img src="./docs/img/vortex-logo.png" alt="Vortex logo" width="200px" height="200px">
-  <h1 align="center">Vortex</h1>
+  <img src="docs/img/vortex-logo.png" alt="Vortex logo" width="200">
+  <h1 align="center">Polaris</h1>
```
Comment (Th3Shadowbroker) [inline README.md:9]:
```suggestion
  <h1 align="center">Vortex</h1>
```

File: README.md [hunk 4]
```diff
@@ -26,8 +32,24 @@ There's a docker-compose file included in this project which provides a Kafka, K
 - A running Kafka broker
 - A running MongoDB instance
 
-## Running Vortex
-To run vortex simply run `vortex serve` to start processing incoming messages.
+## Building Vortex
+
+### Go build
+
+Assuming you have already installed [go](https://go.dev/), simply run the follwoing to build the executable:
```
Comment (Th3Shadowbroker) [inline README.md:39]:
```suggestion
Assuming you have already installed [Go](https://go.dev/), simply run the follwoing to build the executable:
```

File: README.md [hunk 5]
```diff
@@ -53,6 +75,20 @@ Vortex supports configuration via environment variables and/or a configuration f
 | mongo.writeConcern.writes  | VORTEX_MONGO_WRITECONCERN_WRITES  | int           | 1                         | The amount of writes required for a write to be acknowledged. ([See MongoDB docs](https://www.mongodb.com/docs/manual/reference/write-concern/))             |
 | mongo.writeConcern.journal | VORTEX_MONGO_WRITECONCERN_JOURNAL | bool          | false                     | Whether new entries have to be written to disk to be acknowledged or not. ([See MongoDB docs](https://www.mongodb.com/docs/manual/reference/write-concern/)) |
 
+## Running Vortex
+### Locally
+Before you can run Vortex locally you must have a running instance of Kafka and MongoDB locally or forwarded from a remote cluster.
```
Comment (Th3Shadowbroker) [inline README.md:80]:
```suggestion
Before you can run Vortex locally, you must have a running instance of Kafka and MongoDB locally or forwarded from a remote cluster.  
```

**Review by Th3Shadowbroker — DISMISSED** [review]
Please check my suggestions. 
I'd appreciate it if you could have a look as well @ledex 🙂

**Comment by Th3Shadowbroker** [comment]
As this PR only updates one file, I'd suggested that the PR is squashed on merge.

**Review by ledex — COMMENTED** [review]
Appart from @Th3Shadowbroker's suggestions it looks good to me.

**Review by mherwig — COMMENTED** [review]
should be resolved now. Thanks

**Review by chrisingenhaag — APPROVED** [review]

**Review by Th3Shadowbroker — APPROVED** [review]

**Pull request merged**